### PR TITLE
Fix: Outdated discord logo

### PR DESCRIPTION
**Pull request by Mazzya**
## This PR adds/fixes
* Update of the [discord logo](https://github.com/devicons/devicon/issues/907)

### CODE REVIEW

File: docs/assets/css/discord-logo.svg [hunk 1]
```diff
@@ -1 +1,10 @@
...
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="71" height="55" fill="white"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (Panquesito7) [inline docs/assets/css/discord-logo.svg:10]: Awesome! Let's optimize it. 😄
```suggestion
<svg width="71" height="55" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path d="M60.105 4.898A58.55 58.55 0 0 0 45.653.415a.22.22 0 0 0-.233.11 40.784 40.784 0 0 0-1.8 3.697c-5.456-.817-10.886-.817-16.23 0-.485-1.164-1.201-2.587-1.828-3.697a.228.228 0 0 0-.233-.11 58.386 58.386 0 0 0-14.451 4.483.207.207 0 0 0-.095.082C1.578 18.73-.944 32.144.293 45.39a.244.244 0 0 0 .093.167c6.073 4.46 11.955 7.167 17.729 8.962a.23.23 0 0 0 .249-.082 42.08 42.08 0 0 0 3.627-5.9.225.225 0 0 0-.123-.312 38.772 38.772 0 0 1-5.539-2.64.228.228 0 0 1-.022-.378 31.17 31.17 0 0 0 1.1-.862.22.22 0 0 1 .23-.03c11.619 5.304 24.198 5.304 35.68 0a.219.219 0 0 1 .233.027c.356.293.728.586 1.103.865a.228.228 0 0 1-.02.378 36.384 36.384 0 0 1-5.54 2.637.227.227 0 0 0-.121.315 47.249 47.249 0 0 0 3.624 5.897.225.225 0 0 0 .249.084c5.801-1.794 11.684-4.502 17.757-8.961a.228.228 0 0 0 .092-.164c1.48-15.315-2.48-28.618-10.497-40.412a.18.18 0 0 0-.093-.084Zm-36.38 32.427c-3.497 0-6.38-3.211-6.38-7.156 0-3.944 2.827-7.156 6.38-7.156 3.583 0 6.438 3.24 6.382 7.156 0 3.945-2.827 7.156-6.381 7.156Zm23.593 0c-3.498 0-6.38-3.211-6.38-7.156 0-3.944 2.826-7.156 6.38-7.156 3.582 0 6.437 3.24 6.38 7.156 0 3.945-2.798 7.156-6.38 7.156Z" fill="#23272A"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h71v55H0z"/></clipPath></defs></svg>

```

**Review by Panquesito7 — COMMENTED** [review]
Could you please share a screenshot of how this looks on the website? Thanks. 😄

**Comment by Mazzya** [comment]
> Could you please share a screenshot of how this looks on the website? Thanks. 😄

capture

File: docs/assets/css/discord-logo.svg [hunk 2]
```diff
@@ -1 +1,6 @@
...
+</g>
+</svg>
```
Comment (Panquesito7) [inline docs/assets/css/discord-logo.svg:6]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="200" height="200" viewBox="0 0 200 100"><path style="stroke:none;fill-rule:nonzero;fill:#23272a;fill-opacity:1" d="M130.367 10.688A126.341 126.341 0 0 0 99.02.905a.48.48 0 0 0-.504.239c-1.352 2.421-2.852 5.582-3.903 8.066C82.777 7.43 71 7.43 59.406 9.21c-1.05-2.54-2.601-5.645-3.96-8.066a.498.498 0 0 0-.505-.239 126.005 126.005 0 0 0-31.347 9.781.453.453 0 0 0-.207.176C3.422 40.867-2.047 70.133.637 99.035a.53.53 0 0 0 .199.363c13.172 9.73 25.934 15.637 38.457 19.555.2.059.41-.016.539-.18a92.268 92.268 0 0 0 7.867-12.87.494.494 0 0 0-.27-.684c-4.187-1.598-8.175-3.547-12.011-5.758a.497.497 0 0 1-.047-.824 66.385 66.385 0 0 0 2.383-1.883.476.476 0 0 1 .5-.067c25.203 11.575 52.488 11.575 77.394 0a.476.476 0 0 1 .504.063 64.66 64.66 0 0 0 2.391 1.887.499.499 0 0 1-.043.824 78.998 78.998 0 0 1-12.02 5.754.494.494 0 0 0-.261.687 102.785 102.785 0 0 0 7.863 12.864c.121.171.34.246.54.187 12.581-3.918 25.343-9.824 38.515-19.555a.502.502 0 0 0 .199-.355c3.21-33.414-5.379-62.441-22.77-88.172a.38.38 0 0 0-.199-.184Zm-78.906 70.75c-7.586 0-13.84-7.008-13.84-15.614 0-8.605 6.133-15.613 13.84-15.613 7.77 0 13.96 7.07 13.84 15.613 0 8.606-6.13 15.614-13.84 15.614Zm51.172 0c-7.586 0-13.84-7.008-13.84-15.614 0-8.605 6.133-15.613 13.84-15.613 7.77 0 13.96 7.07 13.84 15.613 0 8.606-6.07 15.614-13.84 15.614Zm0 0"/></svg>

```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Thomas-Boi — APPROVED** [review]
The icon looks great! I was a bit hesitant because it looks off-centered in the GitHub viewer. However, it looks good on the website and that's all that matters

**Pull request merged**